python/cuvs_bench/cuvs_bench/split_groundtruth/split_groundtruth.pl

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/perl
```

Comment:
Ummm... do we need Perl code? Or can we use Python? I have zero Perl experience.

Reply:
This is all twice ported code: once from devtechs and once for now, it is what it is for now - until someone has time to refactor to Python.

Reply:
Okay, sounds good.

Reply:
Yeah this code isn’t pretty nor ideal, but it works for now. This was actually originally in the big-Ann-benchmarks competition. As divye mentioned we have a pending todo to migrate it to Python , but it’s low priority since it works as is for now.